src/Raven.Server/Web/System/AdminServerWideHandler.cs

```diff
@@ -87,89 +89,150 @@ public async Task PutServerWideBackupConfigurationCommand()
-                    context.Write(writer, deleteResponse.ToJson());
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
+                    context.Write(writer, putResponse.ToJson());
                     writer.Flush();
                 }
             }
         }
 
-        // Get all server-wide backups -OR- specific task by the task name... 
-        // todo : consider also returning each db specific details for the studio list view here
+        [RavenAction("/admin/configuration/server-wide/backup", "DELETE", AuthorizationStatus.ClusterAdmin)]
+        public async Task DeleteServerWideExternalReplicationConfigurationCommand()
+        {
+            // backward compatibility
+            await DeleteServerWideTaskCommand(OngoingTaskType.Backup);
+        }
+
+        [RavenAction("/admin/configuration/server-wide/task", "DELETE", AuthorizationStatus.ClusterAdmin)]
+        public async Task DeleteServerWideTaskCommand()
+        {
+            var typeAsString = GetStringQueryString("type", required: true);
+
+            if (Enum.TryParse(typeAsString, out OngoingTaskType type) == false)
+                throw new ArgumentException($"{typeAsString} is unknown task type.");
+
+            await DeleteServerWideTaskCommand(type);
+        }
+
         [RavenAction("/admin/configuration/server-wide/backup", "GET", AuthorizationStatus.ClusterAdmin)]
-        public Task GetServerWideBackupConfigurationCommand()
+        public Task GetServerWideBackupConfigurations()
+        {
+            // backward compatibility
+            GetTaskConfigurations(OngoingTaskType.Backup, JsonDeserializationCluster.ServerWideBackupConfiguration);
+            return Task.CompletedTask;
+        }
+
+        [RavenAction("/admin/configuration/server-wide/tasks", "GET", AuthorizationStatus.ClusterAdmin)]
+        public Task GetServerWideTasks()
+        {
+            var typeAsString = GetStringQueryString("type", required: true);
+            if (Enum.TryParse(typeAsString, out OngoingTaskType type) == false)
+                throw new ArgumentException($"{typeAsString} is unknown task type.");
+            
+            Func<BlittableJsonReaderObject, IDynamicJson> converter;
+            switch (type)
+            {
+                case OngoingTaskType.Backup:
+                    converter = JsonDeserializationCluster.ServerWideBackupConfiguration;
+                    break;
+                case OngoingTaskType.Replication:
+                    converter = JsonDeserializationCluster.ServerWideExternalReplication;
+                    break;
+                default:
+                    throw new ArgumentOutOfRangeException($"Task type '{type} isn't suppported");
+            }
+
+            GetTaskConfigurations(type, converter);
+            return Task.CompletedTask;
+        }
+
+        [RavenAction("/admin/configuration/server-wide/tasks-for-studio", "GET", AuthorizationStatus.ClusterAdmin)]
```

Comment:
What is this? We have dedicated Handlers for Studio items.

Reply:
It's getting all of the server-wide tasks